res/photon/index.html

```diff
@@ -225,6 +225,17 @@ <h3 class='photon-title-20'>Photon Message Bar Generic</h3>
             aria-label="Hide the message" title="Hide the message">
           </button>
         </div>
+        <div class="photon-message-bar sized-to-content">
+          <div class="child-section">
```

Comment:
So this fix involves an additional div here. But we are not using that div anywhere in the codebase. Ideally we should be able to find a solution that doesn't involve an additional div. Would you mind trying to figure out a solution that doesn't involve this?

Reply:
> So this fix involves an additional div here. But we are not using that div anywhere in the codebase. Ideally we should be able to find a solution that doesn't involve an additional div. Would you mind trying to figure out a solution that doesn't involve this?

Alright thank you for the clarification @canova

Reply:
Hi, @canova  I was able to find a way around it without using an additional div, please review the new PR. Thank you